__tests__/fixtures/run/escape-args/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "write-args": "node write-args.js"
```

Comment:
Can do `node -p -e "process.argv[2], process.argv.slice(3).join(' ')"` instead and then read the output from stdout. I'd prefer not to write to file system if/when possible.

Reply:
Turns out it should just be `node -p "..."` without the `-e` part.